capio/server/include/handlers/getdents.hpp

```diff
@@ -1,14 +1,15 @@
 #ifndef CAPIO_GETDENTS_HPP
 #define CAPIO_GETDENTS_HPP
 
+#include "posix/utils/env.hpp"
+
 #include <thread>
 
 #include "remote/backend.hpp"
```

Comment:
```suggestion
#include <thread>

#include "posix/utils/env.hpp"

#include "remote/backend.hpp"
```

Local imports go after system ones.